packages/wdio-cli/tests/__fixtures__/array.conf.js

```diff
@@ -0,0 +1,20 @@
+import path from 'path'
```

Comment:
I don't see this file being used anywhere

Reply:
Again #6593.  It doesn't seem to matter what's in the config file I always get the same result, so I stopped developing this line of testing.

Reply:
I have now deleted this file as the contents are not being used.  Rather the return value of getSpecs is being mocked.